testplan/testing/multitest/suite.py

```diff
@@ -382,12 +383,13 @@ def wrapper(function):
         function.__tags__ = copy.deepcopy(tag_dict)
 
         if parameters is not None:  # Empty tuple / dict checks happen later
-
+            function.__display_name__ = name
```

Comment:
Here, you should set `function.__display_name__ = name if name else function.__name__`.

Ideally this wouldn't be a dunder attribute (i.e. just `function._display_name`, however since that is the established convention it's probably best to stick with it for now.

Reply:
make sense